### PR TITLE
Translation file for the 'Romanian' language. Also recognised by ro_RO. Known in romanian as "Română".

### PR DESCRIPTION
**Pull request by 0wnrepo**
Well :-)

### CODE REVIEW

**Comment by nicosomb** [comment]
@j0k3r any idea why Travis failed partially?

File: src/Wallabag/CoreBundle/Resources/translations/messages.ro.yml [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+# Howto
+Form: Formular
+Thanks to this form: "Mulțumită acestui formular"
+Browser addons: "Add-On-uri de Browser"
+Mobile apps: "Aplicații mobile"
+Bookmarklet: "Bookmarklet"
+Standard Firefox Add-On: "Add-On standard de Firefox"
+Chrome Extension: "Extensie Chrome"
+download the application: "descarcă aplicația"
+"Drag &amp; drop this link to your bookmarks bar:": "Drag &ampș drop acest link în bara de bookmark-uri:"
```
Comment (0wnrepo) [inline src/Wallabag/CoreBundle/Resources/translations/messages.ro.yml]: &amp;

**Comment by tcitworld** [comment]
They are passing now.

**Comment by j0k3r** [comment]
@nicosomb sorry I didn't see the problem ..

**Pull request merged**